example_hpc_submission_scripts/australia_pawsey_setonix.sh

```
@@ -0,0 +1,17 @@
+#SBATCH --ntasks=32
+#SBATCH --ntasks-per-node=32
+#SBATCH --cpus-per-task=1
+#SBATCH --account=pawsey0380
+#SBATCH --job-name=TSR_RATTLE_struct_1871_2_Mn_8.sh
+#SBATCH --time=4:00:00
+#SBATCH --partition=work
+#SBATCH --export=NONE
+#SBATCH --mem=32GB
+##SBATCH --exclusive
+module load vasp/5.4.4
+cd "$PBS_O_WORKDIR"
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)